internal/xds/translator/listener.go

```diff
@@ -400,6 +399,16 @@ func buildDownstreamQUICTransportSocket(tlsConfig *ir.TLSConfig) (*corev3.Transp
 			})
 	}
 
+	if tlsConfig.CACertificate != nil {
```

Comment:
consider also sending the client certificate upstream, e.g.  by updating the HCM default for [forwarding](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto#enum-extensions-filters-network-http-connection-manager-v3-httpconnectionmanager-forwardclientcertdetails) to `SANITIZE_SET`. This is not strictly required for the use case in #88, but is a common requirement when a proxy performs client cert auth.

Reply:
I'll raise a separate issue for propagating `XFCC`